application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -169,34 +169,45 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
 
 
         String applicationId;
+        String applicationName;
 
         if (applicationOption != null) {
-            applicationId = VC_APPLICATION_TO_ID.get(applicationOption.getAsString());
+            applicationName = applicationOption.getAsString();
+            applicationId = VC_APPLICATION_TO_ID.get(applicationName);
         } else {
             applicationId = idOption.getAsString();
+            applicationName = applicationId;
```

Comment:
```suggestion
                applicationName = "an activity"
```
Makes more sense than a random number

Reply:
Changed in 16db30d18f0dbe01b7948578f5f9ad451f41c605